json/src/spec/builtin.rs

```diff
@@ -56,17 +45,13 @@ pub struct AltBn128Pairing {
 	pub base: usize,
 	/// Price per point pair.
 	pub pair: usize,
-	/// EIP 1108 transition base price
-	pub eip1108_transition_base: usize,
-	/// EIP 1108 transition price per point pair
-	pub eip1108_transition_pair: usize,
 }
 
 /// Pricing variants.
 #[derive(Debug, PartialEq, Deserialize, Clone)]
 #[serde(deny_unknown_fields)]
 #[serde(rename_all = "snake_case")]
-pub enum Pricing {
+pub enum InnerPricing {
```

Comment:
Bikeshed, but I think the codebase in general prefers `SomethingInner` vs `InnerSomething`.